src/conventions/namespace.md

```diff
@@ -50,3 +58,211 @@ The above examples are my style of namespacing, but there are many other approac
+> **6 Features of Good Software Code**
+> - **Readable** : Would it be better if other developers Read the directory and understand who wrote this data pack from here. 
+> - **Comment** : It would be better if there was
+> - **Uncomplicated** : -
+> - **High flexible** : It is very likely that there will be some part of the command that will need to be altered, refactor, or reused. If other datapacks use your datapack as well, such as core datapack, it will not immediately know who wrote it. 
+> - **Easy maintenance** : That being said, it's useful for workspaces and grouping directories into classes. And it's useful for locating a directory.
+> - **It works exactly as required** : -
+>
+In the features **High flexible & Easy maintenance** In dev datapack circles, there are identical datapacks written by different devs (but not copied at all), with the same functionality. (with different command's algorithm written in different dev methods, but with the same result) and may be slightly different, according to the functionality and design features. 
+> Example cool stuff datapack here
+> - [Veinminer by Boomber](https://www.planetminecraft.com/data-pack/veinminer-by-boomber/)
+> - [VeinMiner by Gnottero](https://www.planetminecraft.com/data-pack/javd-just-another-veinminer-datapack/)
+> - [Vein Miner Unified by Elemend](https://github.com/Elemend/Vein-Miner-Unified)
+
+What is the problem if you use `datapack_name` at `namespace`? Chances are, but very few. But it doesn't mean there isn't. For example, the veinminer datapack, if you set the namespace to vm and other devs use it as well, it's an issue.
+and does not explicitly identify under the developer's name. Because they are not grouped or classes directly under your name.
+```
+data veinminer:main
+ ├ veinminer
+ ├ <global>
+ │    ╰ advancements
+ │        ├ convention.json
+
+data - villager_magic:main
+ ├ villager_magic
+ ├ <global>
+ │    ╰ advancements
+ │        ├ convention.json
+
+data - veinminer_faq:tick
+ ├ veinminer_faq
+ ├ <global>
+ │    ╰ advancements
+ │        ├ faq.json
+
+data - vm:tick
+ ├ vm (Virtual Magic)
+ ├ <global>
+ │    ╰ advancements
+ │        ├ faq.json
+```
+In view of the convention mod, it lacks flexibility and easy maintenance.
+
+ But it's not wrong if you to use `datapack_name` at `namespace`.
+```
+data - convention:veinminer/main
+ ├ convention
+ │    ╰ functions
+ │        ╰ veinminer
+ ├ <global>
+ │    ╰ advancements
+ │        ├ convention.json
+
+data - convention:villager_magic/main
+ ├ convention
+ │    ╰ functions
+ │        ╰ villager_magic
+ ├ <global>
+ │    ╰ advancements
+ │        ├ convention.json
+
+ data - faq:veinminer/tick
+ ├ faq
+ │    ╰ functions
+ │        ╰ veinminer
+ ├ <global>
+ │    ╰ advancements
+ │        ├ faq.json
+
+data - faq:virtual_magic/tick
+ ├ faq
+ │    ╰ functions
+ │        ╰ virtual_magic
+ ├ <global>
+ │    ╰ advancements
+ │        ├ faq.json
+
+ ```
```

Comment:
Clarification needed

Reply:
**Lines 169 - 174** I want to convey is For people using versions lower than 1.18 which has character length limitation. Using a namespace with a creator alias, It's good enough to avoid conflicting scoreboard objectives to some extent.
And there's a slight hint that fake_player doesn't need a namespace, but if you want it to convey clear functionality, you can use the fake_player namespace, since there are no character length limitations.

**Lines 175 - 184** are instructions for using the namespace in that topic.

**Line 187**, I'm going to say that reviewer will continue to review versions below 1.18, but will ignore the namespace conflict in older versions.

**Lines 188 - 189** This is the case of the datapack you reviewed. And you taught me in this matter.

**Lines 190 - 268** I will say that the name of the datapack can still be used as a namespace. If there is an issue that prevents the creator from switching to the creator name at the namespace 
Here I would like to inform them visually that the use of How good is the namespace creator name? In the form of 6 features of good software code? Here, I want to communicate better. In case, it will help them understand more.
And I have made an example, three datapacks are the same. As an example of issues that can arise in the future, I think you understand this, and I think you have some suggestion to convey it better.
And what I want to convey is that, "But it's not wrong to use the name of the datapack as a namespace". Since it is not mandatory and everyone is free to use it however there are no restrictions.
Which I don't forget that "there are many other approaches that you can take, such as `namespace_foo`, `NAMESPACEfoo`, `namespace.foo`, `namespace:foo` and `namespace/foo`. Your imagination is the limit!".